packages/flutter_tools/lib/src/ios/mac.dart

```diff
@@ -462,6 +462,7 @@ Future<XcodeBuildResult> buildXcodeProject({
         await globals.processUtils.run(
           <String>[
             'rsync',
+            '-8',
```

Comment:
Can you add a comment about why we added this?  Staring at the `rsync` man page I don't think it's super obvious why we would use this flag.  Something like:
```suggestion
            '-8', // Avoid mangling filenames with encodings that do not match the current locale.
```